sw/airborne/boards/apogee/chibios/v1.0/board.h

```diff
@@ -205,146 +202,92 @@
 /*
  * PWM defines
  */
-#ifndef USE_PWM0
-#define USE_PWM0 1
-#endif
-#if USE_PWM0
-#define PWM_SERVO_0 0
-#define PWM_SERVO_0_GPIO GPIOB
-#define PWM_SERVO_0_PIN GPIO0
-#define PWM_SERVO_0_AF GPIO_AF2
-#define PWM_SERVO_0_DRIVER PWMD3
-#define PWM_SERVO_0_CHANNEL 2
-#define PWM_SERVO_0_ACTIVE PWM_OUTPUT_ACTIVE_HIGH
-#else
-#define PWM_SERVO_0_ACTIVE PWM_OUTPUT_DISABLED
-#endif
-
```

Comment:
It seems that this part is not matching between cfg and header. CFG seems correct with definition of servo from 0 (without index shift), but header is starting from 1 with shifted index. On Apogee the servos are starting at index 0.

Reply:
Somehow I mess up the apogee always and forgot to run make in the folder. Fixed now!